LetSwift/ApplicationViewControllers/Contact/Views/ContactTextView.swift

```diff
@@ -70,7 +70,8 @@ final class ContactTextView: UITextView, ContactFieldProtocol {
     }
     
     private func setupPlaceholderLabel() {
-        placeholderLabel.attributedText = placeholder?.attributed(withAttributes: typingAttributes).with(color: .placeholder)
+        let mappedAttributes = typingAttributes.mapKeys(NSAttributedStringKey.init(rawValue:))
```

Comment:
Apple decided to introduce `NSAttributedStringKey` structure to replace strings as keys in `attributes` dictionaries. Yet, they apparently forgot to update `typingAttributes` property of `UITextView` to `[NSAttributedStringKey: Any]`, so additional mapping is needed to be done manually